includes/Modules/Analytics_4.php

```diff
@@ -571,7 +601,20 @@ private function handle_provisioning_callback( $account_id, $account_ticket ) {
 			return;
 		}
 
-		$this->get_settings()->merge( array( 'propertyID' => $property->_id ) );
+		$create_time    = isset( $property->createTime ) ? $property->createTime : ''; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+		$create_time_ms = '';
+		if ( $create_time ) {
+			$create_time = new \DateTime( $create_time, new \DateTimeZone( 'UTC' ) );
+			// Convert to Unix timestamp and then to milliseconds.
+			$create_time_ms = (int) ( $create_time->getTimestamp() * 1000 );
+		}
+
+		$this->get_settings()->merge(
+			array(
+				'propertyID'         => $property->_id,
+				'propertyCreateTime' => $create_time_ms,
```

Comment:
This will cause the `propertyCreateTime` not to be saved if GA was already connected by the `pre_update_option` filter above.